### PR TITLE
Change readme comment for including the Überauth plug

### PR DESCRIPTION
**Pull request by pikitgb**

### CODE REVIEW

**Comment by doomspork** [comment]
Hello and welcome @pikitgb :wave: Could you explain why we want to make this change? This is not the standard way to use Ueberauth, I'm not sure this works.

**Comment by AlexJuca** [comment]
@doomspork Shouldn't the pipeline be defined in the ```routes.ex``` file instead of the controller?

**Comment by AlexJuca** [comment]
I believe instead of defining the pipeline in the controller, we could simply call:

```plug Ueberauth``` in the controller.

File: README.md [hunk 1]
```diff
@@ -47,11 +47,13 @@
...
-1.  Include the Überauth plug in your controller:
+1.  Include the Überauth plug in your router:
 
     ```elixir
-    defmodule MyApp.AuthController do
-      use MyApp.Web, :controller
+    defmodule MyApp.Router do
+      use MyApp.Web, :router
+
+      require Ueberauth
```
Comment (doomspork) [inline README.md:56]: I don't believe this require is correct. I don't need this in any of my projects. 
```suggestion
```

**Comment by doomspork** [comment]
@AlexJuca we need to really go back through the Ueberauth documentation and clear this up. Both approaches work and I'm not sure one is more "right" than the other.

Putting it in the Controller is certainly a "cleaner" approach where all the Ueberauth functionality is contained within a single controller.

**Pull request merged**